geth/common/utils.go

```diff
@@ -143,6 +144,11 @@ func Fatalf(reason interface{}, args ...interface{}) {
 
 // CreateTransaction returns a transaction object.
 func CreateTransaction(ctx context.Context, args SendTxArgs) *QueuedTx {
+	if args.Input == nil && args.Data == nil {
+		// If neither Input nor Data were initialized, let's pass an empty slice in Input
+		emptyByteArray := hexutil.Bytes(make([]byte, 0))
+		args.Input = &emptyByteArray
```

Comment:
If `Input` is a pointer why can't it be `nil`?

Reply:
@adambabik the changes to the `SendTxArgs` struct were based on the fact that is looked like a clone of the one in `vendor/github.com/ethereum/go-ethereum/internal/ethapi/api.go`, so I assumed we wanted to keep them in sync (even the comments in our struct are identical). If we have the freedom over our own struct, we should have a comment on it that clarifies that. I'll add one for your review.